README.md

```diff
@@ -191,7 +219,16 @@ Attributes
  * `node["kafka"]["offset_monitor"]["db_name"]` : The base file name for the offset monitoring database file written into the kafka user's home directory (default = "offset_monitor")
  * `node["kafka"]["service"]["stdout"]` : The file to keep std output of kafka init service (default = "/dev/null")
  * `node["kafka"]["service"]["stderr"]` : The file to keep std error of kafka init service (default = "/dev/null")
-
+ * `node["kafka"]["mirror_maker"]["whitelist"]` : A whitelist of topics that will be mirrored. (default = ".*")
+ * `node["kafka"]["mirror_maker"]["blacklist"]` : A blacklist of topics that will not be mirrored. (default = nil)
+ * `node["kafka"]["mirror_maker"]["streams"]` : Number of consumer threads to run. (default = 2)
+ * `node["kafka"]["mirror_maker"]["mirror_target.properties"][*]` : A key/value that will be set in mirror's target properties file. Used to customize the mirror configuration. See [Mirror docs](https://kafka.apache.org/documentation.html#basic_ops_mirror_maker)
+ * `node["kafka"]["mirror_maker"]["mirror_sources"]` : a list of source property files to create. (default = ["mirror_source1.properties"])
```

Comment:
I'll play with this. I'm not in love with the current namespace. 

I did it that way so I could use more-or-less the same logic as the main kafka recipe w/ property file name as the top namespace for items in the file. See: https://github.com/cerner/cerner_kafka/blob/v1.1.0/recipes/default.rb#L198-L212 and https://github.com/PaytmLabs/chef-cerner_kafka/blob/feature-mirrormaker-rebase/recipes/mirror_maker.rb#L50-L64

But if you like the hash I like the presentation better.